frontend/packages/topology/src/components/graph-view/TopologyControlBar.scss

```
@@ -0,0 +1,3 @@
+.odc-topology-control-bar{
+  left: var(--pf-global--spacer--sm)
```

Hey, the css variables top, left, right, bottom are normally used to [position an elements absolute](https://developer.mozilla.org/de/docs/Web/CSS/Left), not to add extra margin or padding. Can you please check if padding-left or margin-left works here as well?

```suggestion
.odc-topology-control-bar {
  padding-left: var(--pf-global--spacer--sm)
```